README.md

```diff
@@ -2,7 +2,7 @@
   <img src="https://cdn.rawgit.com/SwifterSwift/SwifterSwift/master/Assets/logo.svg" title="swifterSwift">
 </p>
 
-[![Build Status](https://api.travis-ci.org/SwifterSwift/SwifterSwift.svg?branch=master)](https://travis-ci.org/SwifterSwift/SwifterSwift)
+[![Build Status](https://github.com/SwifterSwift/SwifterSwift/workflows/SwifterSwift/badge.svg?branch=master)](https://github.com/SwifterSwift/SwifterSwift/actions)
```

Comment:
since its in the same domain, maybe we can make it relative to repository?

```
[![Build Status](/SwifterSwift/SwifterSwift/workflows/SwifterSwift/badge.svg?branch=master)](/SwifterSwift/SwifterSwift/actions)
```

Reply:
> since its in the same domain, maybe we can make it relative to repository?
> 
> ```
> [![Build Status](/SwifterSwift/SwifterSwift/workflows/SwifterSwift/badge.svg?branch=master)](/SwifterSwift/SwifterSwift/actions)
> ```

This might not work in cocoapods.org

Reply:
ah okay ⚡️